src/App.js

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import './App.css';
+// import Missions from "./components/Missons";
```

Comment:
- [ ] Let's not use commented code in your project. This makes code hard to read.